src/index.html

```diff
@@ -11,7 +11,205 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post" 
+      id="form"> 
+      <fieldset class="fieldSet">
+        <legend>Personal Information</legend>
+     
+      
+        <div>
+          <label for="sname">Surname:</label>
+          <input 
+          type="text" 
+          name="surname"
+          id="sname"
+          > <br>
+        </div>
+        
+        <div>
+          <label for="name">Name:</label>
+          <input 
+          type="text" 
+          name="name"
+          id="name"
+          ><br>
+        </div>
+        
+        <div>
+          <label for="age">How old are You?</label>
+          <input 
+          type="number"
+          min="1"
+          max="100"
+          value="12"
+          id="age"
+          ><br>
+        </div>
+        
+        <div>
+          <label for="dobirth">Full date of birth:</label>
+          <input 
+          type="date"
+          id="dobith"
+          name="dobirth"
+          value="yyyy-mm-dd"
+          ><br>
+             
+        </div>
+        
+        <div>
+          <label for="termAgreement">I accept the term of the agreement:</label>
+          <input 
+          type="checkbox"
+          id="termAgreement"
+          name="termAgreement"
+          >
+        </div>  
+         
+      
+
+      </fieldset>
+    
+
+      <fieldset class="fieldSet">
```

Comment:
```suggestion
        </div>  
      </fieldset>   

      <fieldset class="fieldSet">
```
Don't leave newline between parent and child element